modules/brew/brew-setup.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Setup Brew
+Wants=network-online.target
+After=network-online.target
+ConditionPathExists=!/etc/.linuxbrew
+ConditionPathExists=!/var/home/linuxbrew/.linuxbrew
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/mkdir -p /tmp/homebrew
+ExecStart=/usr/bin/tar --zstd -xvf /usr/share/homebrew.tar.zst -C /tmp/homebrew
+ExecStart=/usr/bin/cp -R -n /tmp/homebrew/home/linuxbrew/.linuxbrew /var/home/linuxbrew
```

Comment:
/home/linuxbrew doesn't exist by default. You need to make it prior to this copy. You can either do a mkdir here or use tmpfiles.d

Reply:
It looks like it's being in `tmpfiles.d` already?

https://github.com/blue-build/modules/pull/239/files#diff-a34ba5d080c32ad5ffa035babc519b97dd8596065fcd1b9a3b48e74b79fc4450R3